core/metacling/src/TClingCallFunc.cxx

```diff
@@ -1016,6 +1019,45 @@ tcling_callfunc_Wrapper_t TClingCallFunc::make_wrapper()
    return (tcling_callfunc_Wrapper_t)F;
 }
 
+// FIXME: Sink in the code duplication from get_wrapper_code.
+static std::string PrepareTorWrapper(const Decl* D, const char* wrapper_prefix,
```

Comment:
What's a "tor"? Maybe better:
```suggestion
static std::string PrepareStructorWrapper(const Decl* D, const char* wrapper_prefix,
```
("structor" is the commonly used generalization of ctor+dtor.).